doc/introduction/interfaces.rst

```diff
@@ -6,12 +6,33 @@
 Interfaces
 ==========
 
-PennyLane integrates quantum nodes made up of variational circuits with other programming
-and machine learning frameworks.
-Such frameworks are called *interfaces*. The default interface, implicitly used in the previous section on
-how to program :ref:`variational circuits <intro_vcircuits>`, is NumPy.
+PennyLane offers seamless integration between classical and quantum computations. Code up quantum 
+circuits in PennyLane, and connect them easily to the top scientific computing and machine learning 
+libraries.
 
-Currently, there is support for the following three interfaces:
+The bridge between the quantum and classical worlds is provided in PennyLane via *interfaces*. 
+Currently, there are three built-in interfaces: NumPy, PyTorch, and TensorFlow. 
+These interfaces make each of these libraries quantum-aware, allowing quantum circuits to be
+treated just like any other operation. 
+
+In PennyLane, an interface is declared when creating a :class:`~.QNode`, e.g., 
```

Comment:
Does the QNode link work?

Reply:
locally, yes (but still some issues with css on the navbar)